### PR TITLE
Adds Contact Us Page

### PR DESCRIPTION
**Pull request by watermelonjuice**

### CODE REVIEW

File: core/src/main/resources/runtime-properties/common-shared.properties [hunk 1]
```diff
@@ -16,11 +16,8 @@
 # EMAILS - Settings for system generated emails #
 # ############################################# #
 
-customerService.fromAddress="customerservice@heatclinic.com";
+site.emailAddress=customerservice@heatclinic.com
 
-# Forgot Password Email
-customerForgotPasswordEmail.emailTemplate=/emailTemplates/forgotPassword.html
-customerForgotPasswordEmail.subject=Forgot Password Request - Heat Clinic
```
Comment (phillipuniverse) [inline core/src/main/resources/runtime-properties/common-shared.properties:23]: Looks like these were inadvertently deleted. If you could add these back then I'll get this merged in. Thanks!

**Comment by watermelonjuice** [comment]
@phillipuniverse done

**Comment by phillipuniverse** [comment]
One final thing here, if you could change the branch that this targets to BroadleafCommerce-3.1.x that would be great.

**Comment by phillipuniverse** [comment]
Pretty sure this should be closed in favor of #98

**Pull request closed without merging**